src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -883,8 +912,8 @@ namespace winrt::TerminalApp::implementation
         // `-w -1` will ensure a new window is created.
         winrt::hstring cmdline{
             fmt::format(L"-w -1 new-tab {}",
-                        newTerminalArgs ? newTerminalArgs.ToCommandline().c_str() :
-                                          L"")
+                        newContentArgs ? terminalArgs.ToCommandline().c_str() :

```

Comment:
wait, didn't we already verify that terminalArgs was present before we got here?